generators/client/templates/angular/src/main/webapp/app/admin/audits/_audits.component.html

```diff
@@ -55,9 +56,10 @@ <h4 jhiTranslate="audits.filter.title">Filter by date</h4>
                     <span *ngIf="audit.data" ng-show="audit.data.remoteAddress"><span jhiTranslate="audits.table.data.remoteAddress">Remote Address</span> {{audit.data.remoteAddress}}</span>
                 </td>
             </tr>
+            </tbody>
         </table>
     </div>
-    <div *ngIf="audits">
+    <div>
```

Comment:
why this condition is removed?

Reply:
This condition was useless because already present in the parent `div` (see https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/main/webapp/app/admin/audits/_audits.component.html#L19)